asmcomp/arm/proc.ml

```diff
@@ -224,6 +224,29 @@ let loc_external_results res =
 
 let loc_exn_bucket = phys_reg 0
 
+(* See "DWARF for the ARM architecture" available from developer.arm.com. *)
+
+let int_dwarf_reg_numbers =
+  [| 0; 1; 2; 3; 4; 5; 6; 7; 12 |]
+
+let float_dwarf_reg_numbers =
```

Comment:
For armv6 the registers numbers are 64-95, since this is actually meant with `Legacy VFP v2-use`.

Reply:
I think I've fixed this.